addons/point_of_sale/models/pos_order.py

```diff
@@ -137,7 +137,7 @@ def _prepare_combo_line_uuids(self, order_vals):
             line = line[2]
 
             if line.get('combo_line_ids'):
-                filtered_lines = list(filter(lambda l: l[2].get('id') and l[2].get('id') in line.get('combo_line_ids'), order_vals['lines']))
+                filtered_lines = list(filter(lambda l: l[0] in [0, 1] and l[2].get('id') and l[2].get('id') in line.get('combo_line_ids'), order_vals['lines']))
```

Comment:
Make sense to me :+1: You can open your PR for review. I left a message for the next reviewer.

Data like `[0, 0, {.......}]` or `[1, 0, {.......}]` corresponds to create and update methods. On the contrary when we have something like `[3, 29]` it corresponds to an unlink method. Ad when we look at what `combo_child_uuids_by_parent_uuid = self._prepare_combo_line_uuids(order)` is used for we see that it's used for linking the combo items: `pos_order._link_combo_items(combo_child_uuids_by_parent_uuid)`.

So it make sense to me to not consider the unlink commands, although I wonder if we should include the link and set commands as well in the filter (4 and 6). 
![image](https://github.com/user-attachments/assets/956fbfbb-d640-4bd6-9784-d9014a648fda)

:warning: For the next reviewer, I wasn't able to create a workflow that would show link and set commands. You'll see the bigger picture, do you think those should also be considered in this fix?